src/components/event/EventList/Event-list.js

```diff
@@ -1,46 +1,55 @@
-import React from "react";
+/* eslint-disable no-unused-expressions */
+import React, { useEffect } from "react";
 import { parse as queryStringParse } from "query-string";
 import { connect } from "react-redux";
 import { withRouter } from "react-router-dom";
+
+import PropTypes from "prop-types";
+
 import RenderList from "../RenderList/RenderList";
 import EventCard from "../EventItem/Event-item";
 import filterHelper from "../../helpers/filterHelper";
 import eventStatusEnum from "../../../constants/eventStatusEnum";
-import {
-  reset_events,
-  updateEventsFilters,
-} from "../../../actions/event/event-list-action";
-import { change_event_status } from "../../../actions/event/event-item-view-action";
+import { resetEvents, updateEventsFilters } from "../../../actions/event/event-list-action";
+import { changeEventStatus } from "../../../actions/event/event-item-view-action";
+
```

Comment:
Also please verify extra spaces in different files. There should not be more than 1 empty line between code lines